configuration.nix

```diff
@@ -210,7 +177,13 @@ in makeNetboot {
       environment.etc.host-nix-channel.source = pkgs.path;
     })
 
-    ({pkgs, ...}: {
+    {
+      options.ofborg.package = lib.mkOption {
```

Comment:
a little bit strange that this lib is from outer scope, and why an inline module in imports here instead of putting it directly in the module below?

it does match the surrounding code style, just feels really weird.

Reply:
Yeah all these little modules should probably either be smooshed together or moved into their own files.